ckanext/cfpb_extrafields/plugin.py

```diff
@@ -1,43 +1,173 @@
 import ckan.plugins as p
 import ckan.plugins.toolkit as tk
 import pprint
-
-
 import validators as v
+import options as opts
 
+# if tag usage is going to be expanded, the following should be generalized.
+def create_relevant_governing_documents():
+    user = tk.get_action('get_site_user')({'ignore_auth': True}, {})
+    context = {'user': user['name']}
+    try:
+        data = {'id': 'relevant_governing_documents'}
+        tk.get_action('vocabulary_show')(context, data)
+    except tk.ObjectNotFound:
+        data = {'name': 'relevant_governing_documents'}
+        vocab = tk.get_action('vocabulary_create')(context, data)
+        for tag in opts.relevant_governing_documents(): 
+            data = {'name': tag, 'vocabulary_id': vocab['id']}
+            tk.get_action('tag_create')(context, data)
+def tag_relevant_governing_documents():
+    create_relevant_governing_documents()
+    try:
+        tag_list = tk.get_action('tag_list')
+        tags = tag_list(data_dict={'vocabulary_id': 'relevant_governing_documents'})
+        return tags
+    except tk.ObjectNotFound:
+        return None
+
+def clean_select_multi(a):
+    ''' parses the results of an html form select multiple '''
+    # code review! select multis are contained in unicode strings that look like: 
+    # u'{"blah blah","blah asdf", asdf}' ; u'{asdf, asdf}' ; u'asdf' 
+    # a regex solution would be nice, but for now I demand fields 
+    # do not contain special characters. 
+    # convert to string if a list
+    a = ''.join(a) 
+    # convert back to comma separated array 
+    a = a.split(',') 
+    # kill special characters
+    a = [item.strip('{}"') for item in a]
+    return a
```

Comment:
Will we ever get the value passed in as a list?  If not then we don't need to convert to a string.  Also, we should split at the end after we've filtered out all the unwanted characters instead of running through the list a second time in a list comprehension.  Try this: 
```
return a.replace('{', "").replace("}", "").replace("\"","").split(",")
```